app/_src/gateway/ai-gateway/index.md

```diff
@@ -214,4 +214,33 @@ AI operations, helping you monitor performance and costs effectively.
 
 For more information, see [AI Metrics](/gateway/{{ page.release }}/production/monitoring/ai-metrics).
 
-{% endif_version %}
\ No newline at end of file
+{% endif_version %}
+
+## Quickstart script
+
+Kong provides an interactive AI quickstart script, which launches a demo instance of {{site.base_gateway}} running AI Proxy.
+This script deploys {{site.base_gateway}} in a local Docker container in traditional mode, or as a data plane instance for {{site.konnect_short_name}}.
+
+This option is for demo purposes only, and is not meant for production use.
+
+{:.note}
+> **Note:**
+> Running this script prompts you for AI Provider API Keys which are used to configure authentication with
+> hosted AI providers. These keys are only passed to the {{site.base_gateway}} Docker container and are 
+> not otherwise transmitted outside the host machine.
+
```

Comment:
```suggestion
Kong offers an interactive AI quickstart script that launches a demo instance of {{site.base_gateway}} running AI Proxy. 

```sh
curl -Ls https://get.konghq.com/ai | bash
```

The script will run in either traditional mode or as a data plane instance for {{site.konnect_short_name}}. You will be prompted to input an API key to configure authentication with an AI provider. This key will not be exposed outside of the host machine.
```

Reply:
This rendered poorly, let me try again: 


This is what I mean: 

Kong offers an interactive AI quickstart script that launches a demo instance of {{site.base_gateway}} running AI Proxy. 

```sh
curl -Ls https://get.konghq.com/ai | bash
```

The script will run in either traditional mode or as a data plane instance for {{site.konnect_short_name}}. You will be prompted to input an API key to configure authentication with an AI provider. This key will not be exposed outside of the host machine.